aptible-auth.gemspec

```diff
@@ -22,11 +22,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'aptible-billing', '~> 1.0'
   spec.add_dependency 'aptible-resource', '~> 1.0'
   spec.add_dependency 'gem_config'
-  spec.add_dependency 'oauth2-aptible', '~> 0.10.0'
+  spec.add_dependency 'oauth2', '~> 1.4'
 
-  spec.add_development_dependency 'bundler', '~> 1.3'
+  spec.add_development_dependency 'bundler', '>= 1.3', '< 3.0'
   spec.add_development_dependency 'aptible-tasks', '>= 0.2.0'
-  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rake', '< 11.0'
```

Comment:
If combined with #64, which upgrades to RSpec 3, this can be removed once again. It's only there because certain versions of Rake and certain versions of RSpec 2 don't get along.